sample/src/test/java/app/cash/paparazzi/sample/KeypadViewTestJUnit5.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.sample
+
+import android.animation.ObjectAnimator
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.TextView
+import app.cash.paparazzi.PaparazziTestExtension
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+
+class KeypadViewTestJUnit5 {
+
+  @RegisterExtension
+  val paparazzi = PaparazziTestExtension()
+
+  @Test
```

Comment:
This class is a copy of `KeypadViewTest` with only these lines changed. The `paparazzi` field is now registered with the JUnit 5 test framework and the imported `@Test` annotation is now the JUnit 5/Jupiter version.